### PR TITLE
Feature/prepare with python

### PR DESCRIPTION
**Pull request by sdressler**
Rough implementation of a Python loader. If you all are happy, then I can move TPC-DS and SSB as well.

### CODE REVIEW

File: swarm64_tpc_toolkit/prepare.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        self.vacuum_analyze()
+
+    def prepare_db(self):
+        pass
+
+    def ingest(self, table):
+        return []
+
+    def add_indexes(self):
+# run_if_exists primary-keys.sql
```
Comment (jukoson) [inline swarm64_tpc_toolkit/prepare.py]: commented out

File: prepare_benchmark [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        pass
+
+    benchmark_names = sorted([benchmark.name for benchmark in benchmarks])
+    args_to_parse.add_argument('--benchmark', required=True, choices=benchmark_names, help=(
+        'Which benchmark to run.'
+    ))
+
+    args_to_parse.add_argument('--scale-factor', type=int, default=None,
+                               required=('tpcds', 'tpch', 'ssb') in sys.argv, help=(
+        'Scale factor benchmarks which require one (for instance TPC-DS/H, SSB).'
```
Comment (jukoson) [inline prepare_benchmark]: Sentence seems incomplete. Something like: "Benchmarks TPC-H / TPC-DS .... require a scale factor.

File: prepare_benchmark [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    except FileNotFoundError:
+        pass
+
+    benchmark_names = sorted([benchmark.name for benchmark in benchmarks])
+    args_to_parse.add_argument('--benchmark', required=True, choices=benchmark_names, help=(
+        'Which benchmark to run.'
+    ))
+
+    args_to_parse.add_argument('--scale-factor', type=int, default=None,
+                               required=('tpcds', 'tpch', 'ssb') in sys.argv, help=(
```
Comment (jukoson) [inline prepare_benchmark]: sys.argv could be args.benchmark ?

File: prepare_benchmark (hunk 2 above)
Comment (sdressler) [reply prepare_benchmark]: Yep, I'm short on words today

File: prepare_benchmark (hunk 3 above)
Comment (sdressler) [reply prepare_benchmark]: Not sure, it is not working atm anyways ;-)

**Review by jukoson — COMMENTED** [review]
I would like to test it first early next week

**Comment by jukoson** [comment]
> I would like to test it first early next week

Will do it march 17

**Comment by jukoson** [comment]
@sdressler please resolve the conflict and you have my approval

**Pull request merged**